engine/src/events/loop.cc

```diff
@@ -101,6 +102,7 @@ static void apply_conf(std::atomic<bool>* reloading) {
       std::string path(::config->cfg_main());
       p.parse(path, config);
     }
+    configuration::extended_conf::apply_all_to_state(config);
     configuration::applier::state::instance().apply(config);
```

Comment:
I think I see here why the name.
In fact, you want to update *config* with the extended_conf. Why not calling it 'update_state(...)` in that case?